Packs/Barracuda/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Barracuda Reputation Block List (BRBL)
+- **Breaking Change** ip command changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
This is not a breaking change - the output is the same as it was before, and it doesn't return multiple entries.